charts/global-network-policies/templates/allow-to-seed-apiserver.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: {{ include "networkpolicyversion" . }}
+kind: NetworkPolicy
+metadata:
+  annotations:
+    gardener.cloud/description: |
+      Allows Egress from pods labeled with 'networking.gardener.cloud/to-seed-apiserver=allowed'
+      to Seed's Kubernetes API Server.
+  name: allow-to-seed-apiserver
+  namespace: {{ .Release.Namespace }}
+spec:
+  podSelector:
+    matchLabels:
+      networking.gardener.cloud/to-seed-apiserver: allowed
+  egress:
+  - to:
+    # TODO (mvladev): write a controller which reconciles this policy with
+    # the IP of the 'kubernetes' endpoint in 'default' namespace in the Seed cluster.
+    # - ipBlock:
+    #     cidr: 1.2.3.4/32
+    # TODO (mvladev): We can't be 100% sure what IP the Seed API server might have.
+    # In some cases it might be in the private IP range.
+    - ipBlock:
+        cidr: 0.0.0.0/0
```

Comment:
Doesn't this allow connecting to whatever in the world as soon as some pod is matched by this policy, making all other network policies ineffective?